apps/etterlatte-saksbehandling-ui/client/src/components/tilbakekreving/oppsummering/TilbakekrevingVurderingOppsummering.tsx

```diff
@@ -19,55 +18,53 @@ export function TilbakekrevingVurderingOppsummering({ behandling }: { behandling
   const oppsummertInnkreving = sumNettoTilbakekreving + sumRenter
 
   return (
-    <InnholdPadding>
-      <Table className="table" zebraStripes style={{ width: '40%' }}>
-        <Table.Header>
-          <Table.Row>
-            <Table.HeaderCell></Table.HeaderCell>
-            <Table.HeaderCell>Beregnet feilutbetaling</Table.HeaderCell>
-            <Table.HeaderCell></Table.HeaderCell>
-            <Table.HeaderCell>Tilbakekreving</Table.HeaderCell>
-          </Table.Row>
-        </Table.Header>
-        <Table.Body>
-          <Table.Row key="Beloep">
-            <Table.DataCell></Table.DataCell>
-            <Table.DataCell>{NOK(sumFeilutbetaling)}</Table.DataCell>
-            <Table.DataCell></Table.DataCell>
-            <Table.DataCell></Table.DataCell>
-          </Table.Row>
-          <Table.Row key="Brutto">
-            <Table.DataCell>Brutto tilbakekreving</Table.DataCell>
-            <Table.DataCell></Table.DataCell>
-            <Table.DataCell></Table.DataCell>
-            <Table.DataCell>{NOK(sumBruttoTilbakekreving)}</Table.DataCell>
-          </Table.Row>
-          <Table.Row key="Skatt">
-            <Table.DataCell>Fradrag skatt</Table.DataCell>
-            <Table.DataCell></Table.DataCell>
-            <Table.DataCell>-</Table.DataCell>
-            <Table.DataCell>{NOK(sumSkatt)}</Table.DataCell>
-          </Table.Row>
-          <Table.Row key="Netto">
-            <Table.DataCell>Netto tilbakekreving</Table.DataCell>
-            <Table.DataCell></Table.DataCell>
-            <Table.DataCell>=</Table.DataCell>
-            <Table.DataCell>{NOK(sumNettoTilbakekreving)}</Table.DataCell>
-          </Table.Row>
-          <Table.Row key="Renter">
-            <Table.DataCell>Rentetillegg</Table.DataCell>
-            <Table.DataCell></Table.DataCell>
-            <Table.DataCell>+</Table.DataCell>
-            <Table.DataCell>{NOK(sumRenter)}</Table.DataCell>
-          </Table.Row>
-          <Table.Row key="SumInnkreving">
-            <Table.HeaderCell>Sum til innkreving</Table.HeaderCell>
-            <Table.HeaderCell></Table.HeaderCell>
-            <Table.HeaderCell>=</Table.HeaderCell>
-            <Table.HeaderCell>{NOK(oppsummertInnkreving)}</Table.HeaderCell>
-          </Table.Row>
-        </Table.Body>
-      </Table>
-    </InnholdPadding>
+    <Table className="table" zebraStripes style={{ width: '40%' }}>
```

Comment:
Hvis det er styling her, så putt det heller i en styled component, så opprettholder vi standaren på CSS

Reply:
Når det er en såpass enkel styling med kun ett parameter, er det ikke like greit å ha det inline for å slippe masse styled components?